.github/workflows/prtest.yml

```
@@ -84,34 +58,72 @@ jobs:
+            
+            return { prNumber, username };
+
+
+      - name: Post skip notice
+        if: github.actor == 'dependabot[bot]'
+        uses: actions/github-script@v7
+        with:
+            const username = '${{ fromJSON(steps.pr.outputs.result).username }}';
             const username = '${{ steps.pr.outputs.username }}';
+            await github.rest.issues.createComment({
+              owner: context.repo.owner,
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)